src/Components/test/testassets/BasicTestApp/ServerReliability/JSInterop.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Microsoft.JSInterop;
+
+namespace BasicTestApp
+{
+    public class JSInterop
```

Comment:
We should try to avoid creating a type that overlaps with a prominent namespace.

Reply:
Updated